pkg/config/config.go

```diff
@@ -2147,3 +2151,20 @@ func (c *NetworkConfig) CNIManagerShutdown() {
 func (c *Config) SetSingleConfigPath(singleConfigPath string) {
 	c.singleConfigPath = singleConfigPath
 }
+
+func (c *StatsConfig) Validate() error {
+	// TODO: Because of cyclic dependency, we cannot import the `internal/lib/stats/metrics` package here.
+	availableMetrics := []string{"cpu", "hugetlb", "memory", "network", "oom", "process", "spec", "disk", "diskIO"}
```

Comment:
we could export this here and import it there though. there's a decent amount of  precedent to do that